### DISCUSSION TITLE
posts/2024-07-24-neural-nets/

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#15] [Announcements]
# posts/2024-07-24-neural-nets/

Explainer: what are AI architectures and which are the most important ones?

https://rachel.fast.ai/posts/2024-07-24-neural-nets/

**Comment by arjan-hada** [comment]
You explain things very well Rachel. Thank you for the post. I would love to read your take on T-cell binding prediction. It's on my To-dos.

Here are two of mine on antibody-binding-affinity prediction in the meanwhile.
https://github.com/arjan-hada/antibody-binding-affinity
https://github.com/arjan-hada/esm2-antibody-CLIP

**Reply by racheltho** [reply]
Neat, thanks for sharing the links to your work!  I am writing a literature review of AI applications to immunology and am currently working on a section about antibody design, so this is very relevant :-).

**Reply by arjan-hada** [reply]
Encouragement from someone like you is the fuel on fire for good work. Huge thanks to you and Jeremy for teaching through fastai platform. I am working on my next case study too.
